org.eclipse.xtext.xbase.lib/src/org/eclipse/xtext/xbase/lib/ArrayExtensions.java

```diff
@@ -109,6 +111,29 @@ public static boolean equals(Object[] array, Object other) {
 		return array.equals(other);
 	}
 	
+	 /**
+     * Returns whether the array contains the given element.
+     * 
+     * More formally, returns <tt>true</tt> if and only if this array contains
+     * at least one element <tt>e</tt> such that
+     * <tt>(o==null&nbsp;?&nbsp;e==null&nbsp;:&nbsp;o.equals(e))</tt>.
+     *
+     * @param array
+     * 			the array to test
+     * @param o 
+     * 			element whose presence in this array is to be tested
+     * @return <tt>true</tt> if this array contains the specified element
+     */
+	@Pure
+	public static <T> boolean contains(T[] array, Object o) {
```

Comment:
Unnecessary type parameter, since `T[]` is a subtype of `Object[]`.

Reply:
done